### PR TITLE
smalt - adding recipe for SMALT aligner

### PR DESCRIPTION
**Pull request by notestaff**
* [X] I have read the guidelines above.
* [X] This PR adds a new recipe.  @bioconda/core
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

**Comment by notestaff** [comment]
@bioconda/core

File: recipes/smalt/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    - gcc
+    - autoconf
+    - automake
+    - libtool
+    - pkg-config
+    - python
+    - bambamc
+    - zlib
+  run:
+    - bambamc
```
Comment (bgruening) [inline recipes/smalt/meta.yaml]: zlib also needs to be a runtime dependency

Comment (notestaff) [inline recipes/smalt/meta.yaml]: thanks for catching, added zlib

File: recipes/smalt/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,32 @@
+build:
+  number: 0
+
+package:
+  name: smalt
+  version: "0.7.6"
+source:
+  fn: smalt-0.7.6-static.tar.gz
+  url: https://sourceforge.net/projects/smalt/files/smalt-0.7.6-static.tar.gz/download
```
Comment (bgruening) [inline recipes/smalt/meta.yaml]: the leading `/download` can be removed

File: recipes/smalt/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+  md5: 57ed011836fda8fb5e34604dd1858b5e
+requirements:
+  build:
+    - gcc
+    - autoconf
+    - automake
+    - libtool
+    - pkg-config
+    - python
```
Comment (bgruening) [inline recipes/smalt/meta.yaml]: Is python really needed, couldn't find a reference.

Comment (notestaff) [inline recipes/smalt/meta.yaml]: python 2 is used for testing ("make check") during the build.
this fails on osx so for now i've disabled building there.
i've removed python from runtime dependencies.

**Pull request closed without merging**